models/aws_robomaker_warehouse_GroundB_01/model.sdf

```diff
@@ -23,8 +23,8 @@
         <surface>
           <friction>
             <ode>
-              <mu>0.3</mu>
```

Comment:
@samuelgundry @konduri do we know if this value was calibrated with a certain use-case/robot in mind? if so, would we be breaking that use-case with the new value.

Reply:
There is only one area in our sample apps where we use this actively, at the following app.
 [multi robot sample app](https://github.com/aws-samples/multi-robot-fleet-sample-application)

value of 0.3 causes the husky in multi robot sample app to slip and effectively break, so i can surely give an example of lack of this value causing an issue. 

Our default supported sample app doesn't  run in the warehouse world as seen from the launch files currently supported at 
![image](https://user-images.githubusercontent.com/6376804/103246517-9606c180-4918-11eb-9aa1-3a7b0244c9a6.png)

This app is used by customers and our workshop at https://robomakerworkshops.com/ws/multi_robot_fleet_simulations , so my take is benefits outweigh potential issues.

Reply:
I agree with Vamshi. It makes sense to fix the warehouse for the Robot Fleet sample app. The warehouse world is not used for RoboMaker sample applications elsewhere today.